playbooks/roles/mattermail/defaults/main.yml

```diff
@@ -14,3 +14,6 @@ EMAIL_PASSWORD: set-me-please
 MATTERMAIL_REPO: github.com/open-craft/mattermail
 MATTERMAIL_BRANCH: master
 MATTERMAIL_HOME: /opt/mattermail
+
+# Misc
+MATTERMAIL_MAIL_TEMPLATE: "@here :incoming_envelope: _From: **{{'{{'}}.From{{'}}'}}**_{{'\n'}}>_{{'{{'}}.Subject{{'}}'}}_"
```

Comment:
That's some crazy amount of braces 😄 

Good idea using a variable for helping with further changes.

Reply:
Yeah there are some special characters in there that ansible looks for which were causing issues, so I found we could do it this way. There may be a better way but this works fine.